release-team/role-handbooks/communications/README.md

```diff
@@ -81,26 +81,51 @@ The release lead will drive the content for the release theme and logo.
 
 Tracking, facilitating, and organizing the publication of the Feature Blog series is a major deliverable of the Comms team. Feature blogs are opt-in for SIGs, and are authored by enhancement developers and others close to the features. We do, however, need to encourage owners of important enhancements to opt in to writing feature blogs. Examples of enhancements that warrant a feature blog might include: breaking changes, features and changes important to our users, features that have been in progress for a long time and are graduating, and features that are considered mandatory by the Release Lead. It helps to work closely with the Release Lead and use the respective SIG Slack channels to remind the SIGs about opting in to feature blogs and provide any necessary context to blog authors. 
 
-**The first feature blog** typically goes out on release day alongside or shortly after the release blog, and then are published one-at-a-time, typically at a rate of two to three posts weekly. **The Comms team establishes the publication schedule.**
-> Note that blog PRs in k/website are dated, and automation will publish future-dated entries. This enables a PR process decoupled from blog publication date.
+**Reach out in KEP issues to ask for feature blog opt-in.** Ask every KEP owner if they want to contribute a blog by reaching out in the KEP issue. Example messaging is below.
```

Comment:
I would add a general guideline for reachouts to KEP owners, along these lines:

In case a KEP owner is not being responsive on their GitHub issue (this might be about opt-in, placeholder or addressing review comments), especially near deadlines, it is advised to write a message on the assigned SIG Slack channel possibly quoting the author, whos nickname on Slack would most probably be aligned with the GitHub one.

Do not be afraid of sending a private message too! Many people are in multiple sig channels and they could lose a message or two, but a DM works in the majority of the cases.

In the rare case your messages are not answered within a reasonable amount of time (24-48 hours around deadlnes) you can send an email to the specific mailing list of the sig, consulting with your Comms Lead.

_this last part was suggested by Kat during one of the meetings, I would say it's a last resort but a very powerful one_

Reply:
Resolved!